libs/MobileSync/build.gradle

```diff
@@ -62,3 +67,11 @@ android {
     abortOnError false
   }
 }
+
+ext {
```

Comment:
We need to update setversions.sh [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/setversion.sh#L62) to change the publish version.

Reply:
Will do.  While I'm at it, should I just update the version in `publish-module.gradle` since all of them will always be the same?